src/compiler/scala/tools/nsc/typechecker/RefChecks.scala

```diff
@@ -160,6 +160,72 @@ abstract class RefChecks extends Transform {
         })
       }
     }
+    private def checkDubiousOverloads(clazz: Symbol): Unit = if (settings.warnDubiousOverload) {
+      // nilary members or methods with leading implicit params
```

Comment:
so this comment should say nullary. Just to confirm, we're saying the `NullaryMethodType` is correctly named, and "nilary" was coined so we wouldn't have to rename it.